src/ui/camera.js

```diff
@@ -633,7 +643,11 @@ class Camera extends Evented {
         const zoom = Math.min(tr.scaleZoom(tr.scale * Math.min(scaleX, scaleY)), eOptions.maxZoom);
 
         // Calculate center: apply the zoom, the configured offset, as well as offset that exists as a result of padding.
-        const offset = (typeof eOptions.offset.x === 'number') ? new Point(eOptions.offset.x, eOptions.offset.y) : Point.convert(eOptions.offset);
+        const offset = (typeof eOptions.offset.x === 'number') ?
+            // $FlowFixMe[prop-missing]
+            new Point(eOptions.offset.x, eOptions.offset.y) :
```

Comment:
Nit: Is Flow complaining about `offset.y`? Should we add a check for `y` too to avoid `$FlowFixMe`?